doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.rst

```diff
@@ -1,53 +1,65 @@
 Move Group Interface Tutorial
 ==================================
 
-In MoveIt!, the primary user interface is through the :move_group_interface:`MoveGroup` class. It provides easy to use functionality for most operations that a user may want to carry out, specifically setting joint or pose goals, creating motion plans, moving the robot, adding objects into the environment and attaching/detaching objects from the robot.
+In MoveIt!, the primary user interface is through the :move_group_interface:`MoveGroupInterface` class. It provides easy to use functionality for most operations that a user may want to carry out, specifically setting joint or pose goals, creating motion plans, moving the robot, adding objects into the environment and attaching/detaching objects from the robot. This interface communicates over ROS topics, services, and actions to the `MoveGroup Node <http://docs.ros.org/indigo/api/moveit_ros_move_group/html/annotated.html>`_.
 
-.. tutorial-formatter:: ../move_group_interface_tutorial.cpp
+.. image:: move_group_interface_tutorial_start_screen.png
 
-The entire code
-^^^^^^^^^^^^^^^
-The entire code can be seen :codedir:`here in the moveit_pr2 github project<planning>`.
+Watch the `YouTube video demo <https://youtu.be/4FSmZRQh37Q>`_
 
-Compiling the code
-^^^^^^^^^^^^^^^^^^
-Follow the `instructions for compiling code from source <http://moveit.ros.org/install/>`_.
+Compiling the example code
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+You do not need to build all of MoveIt! from source, but you can follow similar instructions as documented on the  `MoveIt! source install page <http://moveit.ros.org/install/source/>`_ for setting up your catkin workspace. Within your catkin workspace, download this example::
 
-The launch file
-^^^^^^^^^^^^^^^
-The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/planning/launch/move_group_interface_tutorial.launch>`_ on github. All the code in this tutorial can be compiled and run from the moveit_tutorials package
-that you have as part of your MoveIt! setup.
+  git clone https://github.com/ros-planning/moveit_tutorials.git
 
-Running the code
-^^^^^^^^^^^^^^^^
+Temporary PR2 on Kinetic Instructions
+-------------------------------------
 
-Roslaunch the launch file to run the code directly from moveit_tutorials::
+You will also need a **pr2_moveit_config** package to run this tutorial. Currently this is unreleased in ROS Kinetic but the following is a temporary workaround::
 
- roslaunch moveit_tutorials move_group_interface_tutorial.launch
+  git clone https://github.com/PR2/pr2_common.git -b kinetic-devel
+  git clone https://github.com/davetcoleman/pr2_moveit_config.git
```

Comment:
moveit_pr2 has way too many dependencies/custom plugins, and they are not available in kinetic AFAIK. This new moveit_config for the PR2 uses all default MoveIt! components, which is the easiest to maintain and understand. Its up to date with the latest Setup Assistant template files and uses the default KDL solver. Maintenance on the moveit_pr2 repo seems to be lacking and I am not interested in getting involved in that. This new repo was quick and easy for me to setup and allowed me to fix a very broken tutorial. I am happy to move my version elsewhere (e.g. ros-planning org) and eventually switch back to moveit_pr2 if someone else puts in that effort.